test/Npgsql.Tests/Types/TypeHandlerTestBase.cs

```diff
@@ -30,7 +30,7 @@ public async Task Read(string query, T expected)
             using var conn = await OpenConnectionAsync();
             using var cmd = new NpgsqlCommand($"SELECT {query}", conn);
 
-            Assert.AreEqual(await cmd.ExecuteScalarAsync(), expected);
+            Assert.AreEqual(expected, actual: await cmd.ExecuteScalarAsync());
```

Comment:
This is why I prefer NUnit's Assert.That, you never get confused... But am fine either way.

```suggestion
            Assert.That(await cmd.ExecuteScalarAsync(), Is.EqualTo(expected));
```

Reply:
I hope someday you will switch to a better testing framework. Inability to run tests without .NET Core 3 is a major issue with it.

Reply:
How do you mean? What is the inability exactly, I ran tests with .NET Core 3 and now with 5?

Reply:
Recently I uninstalled .NET Core 3 and broke the runner which I didn't know how to fix for a few days. The error told me that there's no assembly for NUnit.

Reply:
Even if you decide to change the testing framework, one assertion won't hurt, so I accepted your suggestion (:

Reply:
I think our CI tests run with only a 5.0 SDK on Github Actions... might be worth a check if this wasn't some other environmental issue.

Yeah, in any case if we ever do decide to change testing frameworks, this assertion is the least of our problems :rofl: IMHO xunit lack many of the nice features NUnit provides (or you have to work a lot to get to something similar).

Reply:
Some things are impossible in NUnit too. @vonzshik even can find a hack arounds in the codebase he maintains at work, but that stuff is easily implementable in XUnit. Sadly, they both are near to be dead/abandoned, so there's no real choice.